### PR TITLE
Update Dependencies in package.json and Switch to pnpm for Onvirtual.Cards

### PR DESCRIPTION
**Pull request by GabCini**
### 📚 Description

Update of the dependencies listed in our package.json file and also switch our package manager from npm to pnpm

----

### ❓ Type of change

_What types of changes does your code introduce? Put an `x` in all the boxes that apply._

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by oleggrech-t — APPROVED** [review]

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
```
Comment (PeterThrels) [inline .husky/pre-commit:1]: confirm this change.

**Review by PeterThrels — APPROVED** [review]

**Review by jasnvella2 — APPROVED** [review]

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# .npmrc
+auto-install-peers=true
+shamefully-hoist=true
+strict-peer-dependencies=true
+shared-workspace-lockfile=false
```
Comment (gaucj-t) [inline .npmrc:5]: ensure these are correct and not copied from other repositories. over here we are not using workspaces.

**Review by gaucj-t — CHANGES_REQUESTED** [review]

**Comment by LucasSaid005** [comment]
Just Tested, found 1 console error.
[Vue warn]: Invalid component name: "layouts/default.vue". Component names should conform to valid custom element name in html5 specification.
image

**Comment by LucasSaid005** [comment]
Found another one:
Uncaught DOMException: Failed to read a named property '__opc_uxsec_11' from 'Window': Blocked a frame with origin "https://js.weavr.io" from accessing a cross-origin frame.
image

File: .npmrc (hunk 2 above)
Comment (GabCini) [reply .npmrc]: I removed `shared-workspace-lockfile=false `

File: .husky/pre-commit (hunk 1 above)
Comment (GabCini) [reply .husky/pre-commit:1]: Most probably this was removed when upgrading the husky dependency

**Comment by gaucj-t** [comment]
> Just Tested, found 1 console error. [Vue warn]: Invalid component name: "layouts/default.vue". Component names should conform to valid custom element name in html5 specification. image
> 
> Edit: This error only came up one time and the layout remained stable (broke nothing)

This error has nothing to do with the dependencies.

**Comment by gaucj-t** [comment]
@GabCini can you also update the node version to be on the latest lts - v20.15.1

**Review by gaucj-t — APPROVED** [review]

**Review by bartolo-maureen — APPROVED** [review]

**Pull request merged**